tox.ini

```diff
@@ -8,7 +8,7 @@ envlist =
     isort
     pylint
     # Python/Django combinations that are officially supported
-    py3{6,7,8,9,10}-django32
+    py3{7,8,9,10}-django32
```

Comment:
Any particular reason for tox now? Figured with github actions, this was moot.

Reply:
Tox is a tool that [belongs to the development layer](https://pythonbiellagroup.gitlab.io/testinginpython/en/tutorial/automazione.html#application-development-and-deployment-automation). It abstracts the details of running tests and linting away. Last but not least, it allows to run all the things locally, without the automation component.

I guess it's here to stay. Is there any good reason to get rid of it?